pkg/kubernetes/status.go

```diff
@@ -45,10 +59,15 @@ func Status() ([]StatusOutput, error) {
 
 	for _, lbl := range controlPlaneLabels {
 		go func(label string) {
-			p, err := ListPods(client, v1.NamespaceAll, map[string]string{
+			defer wg.Done()
+			// Query all namespaces for Dapr pods.
+			p, err := ListPodsInterface(client, map[string]string{
 				"app": label,
 			})
 			if err == nil {
```

Comment:
Since you changed `wg.Done()` to be defer, I think right here you can do
```
if err != nil {
  return
}
```
and unnest the main logic